anchor/eth/src/event_processor.rs

```diff
@@ -604,6 +600,18 @@ impl EventProcessor {
 
         let is_our_validator = self.is_our_validator(&validator_pubkey);
 
+        if !live {
+            if is_our_validator {
+                warn!(
+                    %validator_index,
+                    "Ignoring historic validator exit for validator assigned to us"
+                );
+            } else {
+                debug!(%validator_index, "Ignoring historic validator exit");
```

Comment:
This should be normal, shouldn't it? I'd use debug for our validator and trace otherwise.